docs/RELEASES.md

```diff
@@ -10,7 +10,11 @@ Start by comparing the main branch with the last release in order to fully under
 2. Does the public package API follow all endpoint conventions, such as naming, pointer usage, and options availability? Once these are released, they are permanent in the current major release version.
 3. Are new features generally available in the Terraform Cloud API? Or is there another considered reason to release them?
 
-Ensure CHANGELOG.md is up to date with the next release version before proceeding.
+Make sure CHANGELOG.md is up to date with the next release version before proceeding.
+
+1. Replace `# Unreleased` with the version you are releasing.
```

Comment:
Perhaps clarify that this is a separate process that you PR `before` you release. 

And can we also add language about how to update the dependency if you need the provider to pick up the release immediately, versus when it is scheduled?